build/Linux+BSD/portable/RecipeArm

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# This file is designed to run in debian jessie docker images
+
+main() {
+
+  set -e # Halt on error
+  set -x # Be verbose
+
+  case $1 in
+    "--fetch-dependencies-only")
+      fetchDependencies "$@"
+      ;;
+    "--build-only")
+      shift #need to adjust parameter list so $makefile_overrides start at $1
+      build "$@"
+      ;;
+    "--package-only")
+      package "$@"
+      ;;
+    *)
```

Comment:
I was actually trying to avoid installing wget to keep it as small as possible :).  But if this will make for cleaner more maintainble code, I will go with what you say.  I've actually realized how to avoid the nonense with the forcible apt-get + dpkg install...the trick is to only install the arm versions of dependencies...  Actually what I'll do is use a variable $arch in fetchdependenceis...  stadnby...